src/main/java/net/smartcosmos/dto/metadata/MetadataResponse.java

```diff
@@ -19,26 +21,25 @@
     @Setter(AccessLevel.NONE)
     private int version = VERSION; // just in case there is a default constructor sometime
 
-    private final String dataType;
-    private final String entityReferenceType;
-    private final String key;
-    private final Long lastModifiedTimestamp;
-    private final String rawValue;
-    private final String referenceUrn;
-    private final String urn;
-    private final String moniker;
+    private final String id;
+    private final String ownerType;
+    private final String ownerId;
+    private Map<String, Object> metadata;
+    private final String tenantId;
 
     @Builder
-    @ConstructorProperties({"dataType", "entityReferenceType", "key", "lastModifiedTimestamp", "rawValue", "referenceUrn", "urn", "moniker"})
-    public MetadataResponse(String dataType, String entityReferenceType, String key, Long lastModifiedTimestamp, String rawValue, String referenceUrn, String urn, String moniker) {
-        this.dataType = dataType;
-        this.entityReferenceType = entityReferenceType;
-        this.key = key;
-        this.lastModifiedTimestamp = lastModifiedTimestamp;
-        this.rawValue = rawValue;
-        this.referenceUrn = referenceUrn;
-        this.urn = urn;
-        this.moniker = moniker;
+    @ConstructorProperties({"id", "ownerType", "ownerId", "metadata", "tenantId"})
```

Comment:
I don't think we want public constructors.  I think we want to push toward the Lombok generated code and the `builder()`.

This means `@Builder` moves to the class level and the constructor here is no more.